core/trino-main/src/main/java/io/trino/execution/scheduler/BinPackingNodeAllocatorService.java

```diff
@@ -696,15 +696,15 @@ private synchronized DataSize getEstimatedMemoryUsage(Session session)
 
         private String memoryUsageDistributionInfo()
         {
-            List<Double> quantiles = ImmutableList.of(0.01, 0.05, 0.1, 0.2, 0.5, 0.8, 0.9, 0.95, 0.99);
-            List<Double> values;
+            double[] quantiles = new double[] {0.01, 0.05, 0.1, 0.2, 0.5, 0.8, 0.9, 0.95, 0.99};
+            double[] values;
             synchronized (this) {
                 values = memoryUsageDistribution.valuesAt(quantiles);
             }
 
             return Streams.zip(
-                            quantiles.stream(),
-                            values.stream(),
+                            Doubles.asList(quantiles).stream(),
```

Comment:
hi, what is the advantage of Doubles.asList() vs ImmutableList.of().. It will create a list anyway

Reply:
The main aim here was to remove all existing usages of the list version of `TDigest#valuesAt` (https://github.com/airlift/airlift/pull/1054).
I've also improved this now to avoid `Doubles.asList`